cli/args/mod.rs

```diff
@@ -374,41 +358,6 @@ fn resolve_fmt_options(
   options
 }
 
-#[derive(Clone, Debug)]
-pub struct WorkspaceTestOptions {
-  pub doc: bool,
-  pub no_run: bool,
-  pub fail_fast: Option<NonZeroUsize>,
-  pub permit_no_files: bool,
-  pub filter: Option<String>,
-  pub shuffle: Option<u64>,
-  pub concurrent_jobs: NonZeroUsize,
-  pub trace_leaks: bool,
-  pub reporter: TestReporterConfig,
-  pub junit_path: Option<String>,
-  pub hide_stacktraces: bool,
-}
-
-impl WorkspaceTestOptions {
-  pub fn resolve(test_flags: &TestFlags) -> Self {
-    Self {
-      permit_no_files: test_flags.permit_no_files,
-      concurrent_jobs: test_flags
-        .concurrent_jobs
-        .unwrap_or_else(|| NonZeroUsize::new(1).unwrap()),
-      doc: test_flags.doc,
-      fail_fast: test_flags.fail_fast,
-      filter: test_flags.filter.clone(),
-      no_run: test_flags.no_run,
-      shuffle: test_flags.shuffle,
-      trace_leaks: test_flags.trace_leaks,
-      reporter: test_flags.reporter,
-      junit_path: test_flags.junit_path.clone(),
-      hide_stacktraces: test_flags.hide_stacktraces,
-    }
-  }
-}
```

Comment:
This was supposed to abstract options inferred from flags and `deno.json` configs, but currently just clones over the flags. I've removed it for now. The usage would need to change if it took into account `deno.json`s because it may have to be per-workspace-dir, so the current usage would be confusing to keep.